build_all/make_sdk_cmds_dict.py

```diff
@@ -0,0 +1,6 @@
+output_path = ''
+
+cmds = {
+    "help":         {'short': 'h', 'text': "Print Help text"},
+    "output":       {'short': 'o', 'text': "arg required: Sets the path used to save output folders and libraries."},
+    }
```

Comment:
is this file required? Can we not simplify in one file.

Reply:
It is required. It's a lot easier to have defines in a separate file if you don't want to go through the hastle of making everything class based.